README_InstallCppUTest.txt

```diff
@@ -8,7 +8,9 @@
 2. Build CppUTest and examples
 
 2a. For unix/gcc (including cygwin)
- > cd <someDirectory>/CppUTest/cpputest_build
+ > cd <someDirectory>/CppUTest
+ > ./autogen.sh
+ > cd build
```

Comment:
This shouldn't be there as the packaged version of Cpputest comes with generated configure script. It would need to be changed so that is makes a distinction between the package and the git clone.